pymoab-py3-18.04

```diff
@@ -50,9 +50,10 @@ RUN cd $HOME/opt \
        -DCMAKE_SHARED_LINKER_FLAGS="-Wl,--no-undefined" -DENABLE_MPI=OFF  \
        -DENABLE_HDF5=ON -DHDF5_ROOT=/usr/lib/x86_64-linux-gnu/hdf5/serial \
        -DENABLE_NETCDF=OFF -DENABLE_METIS=OFF -DENABLE_IMESH=OFF -DENABLE_FBIGEOM=OFF \
-       -DENABLE_PYMOAB=ON -DPREFIX=$HOME/opt/moab \
-    && make all -j 8 \
-    && cd .. \
+       -DENABLE_PYMOAB=ON -DCMAKE_INSTALL_PREFIX=$HOME/opt/moab \
+    && make -j 8 \
```

Comment:
Is there a difference between `make all -j 8` and `make -j 8`? Why did you use one in Python 2 and one in Python 3?

Reply:
I actually don't know if there is a difference. I was so focused on trying to get the Python2 version correct, I forgot to translate some changes over to the python 3 version.

Reply:
It did not seem like `make all` included `make install` when I was trying to fix it though

Reply:
FYI - `make -j 8` will make the first target in the Makefile while `make all -j 8` will make a particular target named `all` that may include more/less than the first target.  Often they are the same....